src/ewilgs/urls.py

```diff
@@ -1,11 +1,13 @@
 """API urls"""
 from django.urls import path
 
-from .views import home, get_downlink_table, get_uplink_table, add_downlink_frames
+from . import views
 
 urlpatterns = [
-    path('ewilgs/home/', home, name='home'),
-    path('ewilgs/downlink/', get_downlink_table, name='ewilgs_downlink'),
-    path('ewilgs/uplink/', get_uplink_table, name='ewilgs_uplink'),
-    path('submit/', add_downlink_frames, name='add_downlink_frames'),
+    path('ewilgs/home/', views.home, name='home'),
+    path('ewilgs/downlink/', views.get_downlink_table, name='ewilgs_downlink'),
+    # path('ewilgs/uplink/', views.get_uplink_table, name='ewilgs_uplink'),
```

Comment:
Remove if it's unused

Reply:
Good point! However, this will be added later when uplink view is ready. I wanted to hide it for the first release.